README.md

```diff
@@ -35,9 +35,9 @@ are unsure, it is recommended to use the latest version of `mysql-connector-java
 
 | JDBC Driver Version        | Cloud SQL Socket Factory Version         |
 | -------------------------- | ---------------------------------------- |
```

Comment:
Is it useful to adjust the length of all rows to be the same?

Reply:
Only in that it looks nice in raw MD format